modules/microshift-adding-other-services-to-blueprint.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-update-rpms-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-adding-other-services-to-blueprint_{context}"]
+== Adding other services to a blueprint
+
+After you install {microshift-short} and optional RPMs, the additional packages can be added to your OSTree blueprint to enable them.
+
+.Prerequisites
+
+* An Image Builder blueprint file exists.
```

Comment:
"exists" or was installed? I think we need to make it more explicit that we're talking about installing microshift-<optional-packare>-release-info RPMs.

Reply:
Step one is editing the TOML file, so the prereq is that you have created one.